charts/cht-chart-4x/templates/couchdb-n-deployment.yaml

```diff
@@ -84,9 +84,17 @@ spec:
         volumeMounts:
         - mountPath: /opt/couchdb/data
           name: couchdb-{{ $nodeNumber }}-claim0
-          {{- if ne .Values.couchdb_data.dataPathOnDiskForCouchDB "" }}
-          subPath: {{ .Values.couchdb_data.dataPathOnDiskForCouchDB }}
+          {{- if $.Values.couchdb_data.dataPathOnDiskForCouchDB }}
+          subPath: {{ if contains "%d" $.Values.couchdb_data.dataPathOnDiskForCouchDB }}
```

Comment:
I believe this also parses `data%directory` and may mess up the setup if `data%directory` is provided and is a valid name?

Reply:
Also making a note that across projects these are generally named like:
`data` or `/srv/storage/medic-core/couchdb/data`. The support we're introducing now is for when people setup clustered setup with k3s or docker-compose and have named the directories for individual couchdbs like `/srv1/couchdb/data` (For couchdb1), `srv2/couchdb/data` (For couchdb2) and `srv3/couchdb/data` (For couchdb3).